cmd/oomhero/main.go

```diff
@@ -17,8 +17,24 @@ var (
 
 // reads warning and critical from environment or use the default ones.
 func init() {
-	warning = envVarToUint64("WARNING", warning)
-	critical = envVarToUint64("CRITICAL", critical)
+	warningEnv := envVarToUint64("WARNING", warning)
+	criticalEnv := envVarToUint64("CRITICAL", critical)
+
+	// validate warning and critical https://github.com/ricardomaraschini/oomhero/issues/4
```

Comment:
I think it is safe to remove this comment.

Reply:
removed